### DISCUSSION TITLE
Logging Newbie question

### DISCUSSION THREAD

**Discussion by pbix** [blakeblackshear/frigate#2241] [General]
I am running frigate in a docker container and things are coming along.  I have not yet got to linking things to Home Assistant so at the moment we are running on an x86 box (Linux) and under docker.

I have read all the docs and I see that there is a logging feature and many users have posted examples of their logs.  The website is up, the cameras are more are less working

But I cannot find the location of this log file.  So sorry for this complete noob question but it where are they and how can I view them?  Also a little discussion in the docs of this topic would be great as well.

**Comment by blakeblackshear** [comment]
Logs are handled as a part of docker. For example, if you ran via the example in the docs, you can see the logs with `docker logs -f frigate`. More detailed info for docker logs can be found here: https://docs.docker.com/engine/reference/commandline/logs/

**Comment by pbix** [comment]
Thanks so much for the help.  Frigate really looks nice, thanks for your efforts.
